plugins/chunter-resources/src/components/Message.svelte

```diff
@@ -59,9 +59,32 @@
         action: chunter.actionImpl.SubscribeMessage
       } as Action)
 
+  const pinnedQuery = createQuery()
+  let pinned = false
+
+  pinnedQuery.query(
+    chunter.class.Channel,
+    { _id: message.space },
+    (res) => {
+      pinned = !!res[0]?.pinned?.includes(message._id)
```

Comment:
too many queries, let's move it to ChannelView and ThreadView, and pass  `export let pinned: boolean = false` to this component. Currently you one query for PinnedMessages, and query for each! message